.github/workflows/ci.yml

```diff
@@ -39,35 +53,48 @@ jobs:
         run: go vet github.com/couchbase/sync_gateway/...
 
   test:
+    needs: run-bootstrap
     env:
       GO111MODULE: off
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            set-gopath: echo "GOPATH=$GITHUB_WORKSPACE/godeps" >> $GITHUB_ENV
+
+          - os: windows-latest
+            set-gopath: echo "GOPATH=$Env:GITHUB_WORKSPACE\godeps" >> $Env:GITHUB_ENV
```

Comment:
I know its not really in the scope of this PR but maybe just for completeness we could add macOS? Fairly sure it'd be the same steps as ubuntu but as we already have the matrix setup it should be easy and the tests wouldn't take any longer as they're in parallel.

Just means we'd be covering all the supported github actions OSs for completeness.

Should just be `macos-latest`. Maybe just stick it in, if its not the same as ubuntu don't spend time on it.